dashboards/jenkins/jenkins-prometheus.json

```diff
@@ -84,7 +84,7 @@
           "scorecard": {
             "blankView": {},
             "timeSeriesQuery": {
-              "prometheusQuery": "jenkins_executor_free_value{${Cluster},${Location},${Namespace}}"
+              "prometheusQuery": "sum(jenkins_executor_free_value)"
```

Comment:
We want to have both the sum and the template variable, correct?

If someone filters down to a specific cluster within the GCP project, we want this to calculate `sum(jenkins_executor_free_value)` for all jenkins workloads within the cluster? Same goes for the other gauges in this commit

Reply:
Bump -- do the gauges need to be
- `sum(jenkins_executor_free_value{${Cluster},${Location},${Namespace}})`
- `sum(jenkins_executor_in_use_value{${Cluster},${Location},${Namespace}})`
- `sum(jenkins_plugins_failed{${Cluster},${Location},${Namespace}})`

for the filtering on the template variables to work?